ragstack-e2e-tests/pyproject.llamaindex.toml

```diff
@@ -20,7 +20,7 @@ pillow = "^10.2.0"
 testcontainers = "^3.7.1"
 python-dotenv = "^1.0.1"
 trulens-eval = "^0.21.0"
-nemoguardrails = "^0.7.1"
```

Comment:
did you mean to make this update too?

Reply:
Yeah, this should've been done in a previous PR